src/app/components/ErrorFormatter/index.tsx

```diff
@@ -68,6 +74,14 @@ export function ErrorFormatter(props: Props) {
     [WalletErrors.LedgerUnknownError]: t('errors.unknownLedgerError', 'Unknown ledger error: {{message}}', {
       message,
     }),
+    [WalletErrors.IndexerAPIError]: t(
+      'errors.indexerAPIError',
+      '{{indexerName}} API appears to be down, so some information may be missing or out of date. Please come back later.',
```

Comment:
@tjanez @peterjgilbert thoughts on the text? (see screenshots in PR description)

Reply:
In the future, we should also add a link to our external status dashboard and instruct users to check there for up to date info on outages.